src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/StoreCoverage.java

```diff
@@ -76,6 +76,7 @@ public URL execute(
             } catch (Exception e) {
                 // we tried, no need to fuss around this one
             }
+            os.close();
```

Comment:
@NielsCharlier, perhaps use try-with-resources? Unfortunately GeoTiffWriter does not yet implement AutoCloseable. @aaime?